cranelift/codegen/src/isa/aarch64/inst/emit.rs

```diff
@@ -1275,6 +1275,16 @@ impl<O: MachSectionOutput> MachInstEmit<O> for Inst {
                 sink.add_reloc(srcloc, Reloc::Abs8, name, offset);
                 sink.put8(0);
             }
+            &Inst::GetPinnedReg { rd } => {
+                // Encoded as ORR rd, rm, zero.
+                let rm = xreg(PINNED_REG);
+                sink.put4(enc_arith_rrr(0b10101010_000, 0b000_000, rd, zero_reg(), rm));
```

Comment:
Can we just create an `Inst::Move` and then `inst.emit(sink);` as we do for the other pseudoinstructions?